src/common/DataConversionFlowGraph.cpp

```diff
@@ -226,7 +226,7 @@ Result DataConversionFlowGraph::configure(AudioStream *sourceStream, AudioStream
             mSink = std::make_unique<SinkI32>(sinkChannelCount);
             break;
         default:
-            LOGE("%s() Unsupported sink format = %d", __func__, sinkFormat);
+            LOGE("%s() Unsupported sink format = %s", __func__, oboe::convertToText(sinkFormat));
```

Comment:
In this case, we want to know what strange integer value was returned. So the convertToText() will probably not handle it. So better to cast to (int) and print with %d.